src/utils/rememberMessages.js

```diff
@@ -5,7 +5,7 @@ let rememberedMessages = []
 const addMessage = (message) => { rememberedMessages.push(message);}
```

Comment:
This is a good way to manage this state; simple and built on classic principles. So good work team.